RockWeb/Plugins/org_lakepointe/CheckIn/Labels/Child_Label_and_Slips_v2.lava

```diff
@@ -101,12 +102,16 @@ CT~~CD,~CC^~CT~
 
 ^FO560,340^FB290,1,0,C^A0N,60,60^FD{{ Person.SecurityCode }}^FS
 
-^FO7,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personAllergy != '' %}A{% endif %}^FS
-^FO73,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personMedicalAlert  != '' %}M{% endif %}^FS
-^FO140,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personLegalNotes != '' %}C{% endif %}^FS
-^FO207,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personBaptized  != ''  %}B{% endif %}^FS
-^FO273,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personFoster == 'true' %}P{% endif %}^FS
-^FO340,363^FB67,1,0,L^A0N,43,43^FH\^FD{% if personSelfRelease == 'Yes' and isSelfReleaseEligible == true %}S{% endif %}^FS
+^FO10,355^FB67,1,0,L^A0N,43,43^FH\^FD{% if personAllergy != '' %}A{% endif %}^FS
+^FO70,355^FB67,1,0,L^A0N,43,43^FH\^FD{% if personMedicalAlert  != '' or personMedicalAlert2 == 'Yes' %}M{% endif %}^FS
```

Comment:
New attribute being looked at here is a boolean: 
![image](https://github.com/user-attachments/assets/3644f300-bae2-49ab-b54e-e7307cf68251)